### DISCUSSION TITLE
The new home for GitHub Codespaces beta feedback

### DISCUSSION THREAD

**Discussion by bailey** [community/community#3384] [Codespaces] [answered]
### The GitHub Codespaces Discussions category is now LIVE! 

With GitHub Discussions, you can share suggestions for how products should be improved and discuss those improvements with the community, including members of the GitHub product team. We on the Codespaces team love hearing your feedback and are eager to continue in conversations + knowledge-sharing with you.

Welcome to our new space! 🎉

**Comment by PranamBhat** [comment]
**Wow!! Great going team!!!**

**Comment by 2percentsilk** [comment, marked as answer]
Piling on here 😄  - The team is  excited to hear about your Codespaces experience, learn what features you'd love to see next, and get feedback on ideas we're considering 🤔  Feel free to share the great, the needs-work, and the I-need-this-now to help us make your Codespaces experience ✨shine✨

**Reply by PranamBhat** [reply]
Are you testing something?! 👯 

Previously it was "**#**" icon for "**Codespaces**". Now "laptop" icon came 🥇

**Comment by PranamBhat** [comment]
Hi @bailey @2percentsilk 

It would be great if we can redirect to "**Codespaces Feedback**" category while clicking on this page(if you give hyperlink):


![image](https://user-images.githubusercontent.com/32120033/108305865-f3104c80-71d0-11eb-9a0c-999323f210db.png)

**Reply by bailey** [reply]
Hi there @PranamBhat, this is actually feedback for GitHub Discussions. I'd recommend navigating to the [Discussions category](https://github.com/github/feedback/discussions/categories/discussions-feedback), where you can open up a new discussion with this feedback so that it gets to the right team. Thanks!

**Reply by PranamBhat** [reply]
@bailey  Sure. Thank you.

**Comment by MiguelRipoll23** [comment]
I wish I could submit feedback but I’m not part of the beta 😅 (👉👈)

**Comment by munderseth** [comment]
I have been exploring and playing around with Codespaces. IMHO I think it is going to be wonderful for my use cases. FYI I used Cloud 9 IDE in the early days, which is now apart of AWS - [AWS Cloud9 environment](https://docs.aws.amazon.com/cloud9/index.html).

I wanted to know what the plan is for supporting Juypter Notebooks? It would be great if this functionality was fully supported.

When I create a notebook file: `example.ipynb` it does not work. 

I get the following:

![image](https://user-images.githubusercontent.com/1229367/108533632-0a613f80-728e-11eb-9466-a72f4e2cded5.png)

**Reply by 2percentsilk** [reply]
@munderseth mind opening a new Discussion in the Codespaces category on this topic? We'll be locking this thread since it's just a welcome post and I don't want you to lose the ability to chat with us 😄
